.github/workflows/auto-assign.yaml

```diff
@@ -8,9 +8,8 @@ permissions:
 
 jobs:
   assign:
-    permissions:
+    permissions: write-all
       # write permissions are needed to assign the issue.
-      contents: write
```

Comment:
Should this be `contents: write-all`? Or the `write-all` has a different format on the same line?

Reply:
@travisn , I was wondering the same but  from the doc

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/assigning-permissions-to-jobs#defining-access-for-the-github_token-permissions

it seems like `permissions:write-all` is correct.

Reply:
Ah looks good. Before we try the write-all permissions, how about this?
```
issues: write
```

Reply:
@travisn  wrote: 
> Ah looks good. Before we try the write-all permissions, how about this?
> 
> ```
> issues: write
> ```

good idea, trying

Reply:
done, updated accordingly